net/cryfs/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+PortGroup                 cmake 1.1
+PortGroup                 active_variants 1.1
+
+github.setup              cryfs cryfs 0.10.2
+github.tarball_from       releases
+
+license                   LGPL-3
+
+categories                net fuse
+platforms                 darwin
+
+maintainers               @jolan78 openmaintainer
```

Comment:
This syntax is incorrect. You need to include your obfuscated email address so you can be alerted to tickets and such.

Reply:
Ok, I thought github usernames was used for tickets.

Reply:
Tickets are maintained on trac.macports.org